src/Storages/ya.make

```diff
@@ -182,6 +182,8 @@ SRCS(
     System/StorageSystemTableEngines.cpp
     System/StorageSystemTableFunctions.cpp
     System/StorageSystemTables.cpp
+    System/StorageSystemTimeZones.cpp
+    System/StorageSystemTimeZones.generated.cpp
```

Comment:
Actually ya.make is used for "alien build system" that are using timezones in completely different way... I will exclude system.timezones for that build.